backend/src/main/java/org/sefglobal/core/controller/CertificateController.java

```diff
@@ -20,8 +21,8 @@ public CertificateController(CertificateService certificateService) {
 
     @GetMapping
     @ResponseStatus(HttpStatus.OK)
-    public List<Certificate> getCertificates(){
-        return certificateService.getCertificates();
+    public Page<Certificate> getCertificates(@RequestParam Integer pageNumber, Integer pageSize){
```

Comment:
Any reason to use `Integer` here, if none please use primitive types

Reply:
No particular reason, I'll change it.

Reply:
Done @YohanAvishke